spec/factories/appliance_types.rb

```diff
@@ -23,6 +23,13 @@
       virtual_machine_templates { [ build(:virtual_machine_template)] }
     end
 
+    # If no os_family has been specified by the requestor, bind this appliance to the first available os_family
+    after(:build) do |atype|
+      if atype.os_family.blank?
+        atype.os_family = Atmosphere::OSFamily.first
```

Comment:
If this is required always than `OSFamily` creation should be also added into `seed.rb` in every project which uses atmosphere. `rails generate atmosphere:install` task is a good place to add code responsible for adding additional seeds.

What is more if `OSFamily` is required than maybe we should have validation to not allow to remove all of them from db?